wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -47,14 +47,72 @@ When adding redirects for a new or existing article, keep in mind that they shou
 
 The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for various common errors. The list of checks is configured in the [`continuous-integration.yml`](https://github.com/ppy/osu-wiki/blob/master/.github/workflows/continuous-integration.yml) file. 
 
-The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all plugins used by the CI, from which some were written by the osu! wiki maintainers.
+The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all (remark) plugins used by the CI, of which some were written by the osu! wiki maintainers.
```

Comment:
"all (remark) plugins" reads weird for one reason or another, maybe because it's not quite straightforward it's a tool name

Reply:
would it be better to link it? most of this article is [signposted as being technical](https://github.com/ppy/osu-wiki/blob/89e54bd9cea940104d678ce060343535e23eaee2/wiki/osu!_wiki/Maintenance/en.md?plain=1#L15) anyway

```suggestion
The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all ([remark](https://github.com/remarkjs/remark)) plugins used by the CI, of which some were written by the osu! wiki maintainers.
```

Reply:
yes that gives enough context